### PR TITLE
Re-running GLM-hierarchical-binomial-model.ipynb

### PR DESCRIPTION
**Pull request by percevalve**
# GLM-hierarchical-binomial-model.

Minimal changes, just removing the initial PyMC version print statement.

### CODE REVIEW

File: myst_nbs/generalized_linear_models/GLM-hierarchical-binomial-model.myst.md [hunk 1]
```diff
@@ -6,37 +6,35 @@ jupytext:
...
-  display_name: Python PyMC3 (Dev)
+  display_name: Python 3 (ipykernel)
   language: python
   name: python3
 ---
 
 # Hierarchical Binomial Model: Rat Tumor Example
 :::{post} Nov 11, 2021
-:tags: generalized linear model, hierarchical model, pymc3.Beta, pymc3.Binomial, pymc3.Data, pymc3.Deterministic, pymc3.HalfNormal, pymc3.Model, pymc3.Potential
+:tags: generalized linear model, hierarchical model, pymc.Beta, pymc.Binomial, pymc.Data, pymc.Deterministic, pymc.HalfNormal, pymc.Model, pymc.Potential
```
Comment (OriolAbril) [inline myst_nbs/generalized_linear_models/GLM-hierarchical-binomial-model.myst.md:16]: All the `pymc.` tags should be deleted, not updated from pymc3 to pymc.

**Review by OriolAbril — COMMENTED** [review]
looked at tags only for now

**Comment by twiecki** [comment]
Other than that the code looks fine :+1:.

File: myst_nbs/generalized_linear_models/GLM-hierarchical-binomial-model.myst.md [hunk 2]
```diff
@@ -6,37 +6,34 @@ jupytext:
...
 kernelspec:
-  display_name: Python PyMC3 (Dev)
+  display_name: Python 3 (ipykernel)
   language: python
   name: python3
 ---
 
 # Hierarchical Binomial Model: Rat Tumor Example
 :::{post} Nov 11, 2021
-:tags: generalized linear model, hierarchical model, pymc3.Beta, pymc3.Binomial, pymc3.Data, pymc3.Deterministic, pymc3.HalfNormal, pymc3.Model, pymc3.Potential
```
Comment (twiecki) [inline myst_nbs/generalized_linear_models/GLM-hierarchical-binomial-model.myst.md:16]: @OriolAbril meant to only remove those that started with `pymc.`, not the first 3.

**Review by twiecki — APPROVED** [review]

**Comment by twiecki** [comment]
Thanks @percevalve!

**Pull request merged**